packages/design-system/src/custom/KBar/KBarSearchResults.tsx

```diff
@@ -1,21 +0,0 @@
-import { LayoutGroup } from 'framer-motion'
-import { KBarResults, useMatches } from 'kbar'
-
-import { KBarSearchResult } from './KBarSearchResult'
-
-const KBarSearchResults = () => {
-  const { results } = useMatches()
-
-  return (
-    <>
-      {/* @todo(react-18) */}
-      {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
-      {/* @ts-ignore */}
-      <LayoutGroup id={`kbar-layout-group`}>
```

Comment:
This assisted with nice styling for background transitions.

Reply:
Aye, not going to bother with this right now. Since this is a bit more dynamic than all the preload stuff, may be a bit harder to implement and can come later.